### PR TITLE
Automate course discussions dashboard screen

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-8487](https://openedx.atlassian.net/browse/LEARNER-8487)

Verify that the Course Discussions Dashboard tab will show the following contents,
        Header contents,
            Back icon,
            Discussions as Title,
        Verify that the user should be able to view these Course contents:
            Search posts,
            All posts,
            Posts I'm following,
            General
        Verify all screen contents have their default values
Verify on tapping All Posts will load its screen
         Tapping Posts I'm following will load its screen
         Tapping General will load its screen
          Searching any post in the search bar will load the results screen

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR.?

File: tests/common/globals.py [hunk 1]
```diff
@@ -714,6 +714,22 @@ def get_all_elements_by_id(self, driver, element_to_wait_for):
...
+        wait for element visibility on screen
+        Returns:
+            webdriver element from screen by ID of given index
+        """
+
+        self.wait_for_element_visibility(
+            driver,
+            element_to_wait_for
+        )
+        return self.get_all_elements_by_id(
```
Comment (jawad-khan) [inline tests/common/globals.py:728]: make return statement simple.

File: tests/android/tests/test_android_discussions_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        android_my_courses_list_page = AndroidMyCoursesList(set_capabilities, setup_logging)
+        android_main_dashboard_page = AndroidMainDashboard(set_capabilities, setup_logging)
+
+        assert android_main_dashboard_page.load_courses_tab()
+        if android_my_courses_list_page.get_my_courses_list_row():
+            android_my_courses_list_page.get_first_course().click()
+        else:
+            setup_logging.info('No course enrolled by this user.')
+
+        assert android_course_dashboard_page.get_navigation_icon().get_attribute('content-desc') \
```
Comment (jawad-khan) [inline tests/android/tests/test_android_discussions_dashboard.py:47]: can we beak this line?

File: tests/common/globals.py (hunk 1 above)
Comment (bilalawan321) [reply tests/common/globals.py:728]: done

File: tests/android/tests/test_android_discussions_dashboard.py (hunk 2 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_discussions_dashboard.py:47]: fixed

**Review by jawad-khan — APPROVED** [review]

**Pull request merged**